### PR TITLE
quickstarts code fix

### PR DESCRIPTION
**Pull request by rogerogers**
Signed-off-by: rogerogers <rogers@rogerogers.com>

Thank you for helping make the Dapr documentation better!

**Please follow this checklist before submitting:**
- [x] Commits are signed with Developer Certificate of Origin (DCO - [learn more](https://docs.dapr.io/contributing/contributing-overview/#developer-certificate-of-origin-signing-your-work))
- [x] [Read the contribution guide](https://docs.dapr.io/contributing/docs-contrib/contributing-docs/)
- [x] Commands include options for Linux, MacOS, and Windows within codetabs
- [x] New file and folder names are globally unique
- [x] Page references use shortcodes instead of markdown or URL links
- [x] Images use HTML style and have alternative text
- [x] Places where multiple code/command options are given have codetabs

In addition, please fill out the following to help reviewers understand this pull request:

## Description

code indentation and some command fix

### CODE REVIEW

File: daprdocs/content/en/getting-started/quickstarts/pubsub-quickstart.md [hunk 1]
```diff
@@ -700,13 +700,13 @@ cd pub_sub/go/sdk/order-processor
 Install the dependencies and build the application:
 
 ```bash
-go build
+go build app.go
```
Comment (hhunter-ms) [inline daprdocs/content/en/getting-started/quickstarts/pubsub-quickstart.md]:
```suggestion
go build .
```

File: daprdocs/content/en/getting-started/quickstarts/pubsub-quickstart.md [hunk 2]
```diff
@@ -700,13 +700,13 @@ cd pub_sub/go/sdk/order-processor
...
 ```
 
 Run the `order-processor` subscriber service alongside a Dapr sidecar.
 
 ```bash
-dapr run --app-port 6001 --app-id order-processor --app-protocol http --dapr-http-port 3501 --components-path ../../../components -- go run
+dapr run --app-port 6001 --app-id order-processor --app-protocol http --dapr-http-port 3501 --components-path ../../../components -- go run app.go
```
Comment (hhunter-ms) [inline daprdocs/content/en/getting-started/quickstarts/pubsub-quickstart.md]:
```suggestion
dapr run --app-port 6002 --app-id order-processor-sdk --app-protocol http --dapr-http-port 3501 --components-path ../../../components -- go run .
```

File: daprdocs/content/en/getting-started/quickstarts/secrets-quickstart.md [hunk 3]
```diff
@@ -488,13 +488,13 @@ cd secrets_management/go/sdk/order-processor
 Install the dependencies:
 
 ```bash
-go build
+go build app.go
```
Comment (hhunter-ms) [inline daprdocs/content/en/getting-started/quickstarts/secrets-quickstart.md]:
```suggestion
go build .
```

File: daprdocs/content/en/getting-started/quickstarts/secrets-quickstart.md [hunk 4]
```diff
@@ -488,13 +488,13 @@ cd secrets_management/go/sdk/order-processor
...
 ```
 
 Run the `order-processor` service alongside a Dapr sidecar.
 
 ```bash
-dapr run --app-id order-processor --components-path ../../../components/ -- go run
+dapr run --app-id order-processor --components-path ../../../components/ -- go run app.go
```
Comment (hhunter-ms) [inline daprdocs/content/en/getting-started/quickstarts/secrets-quickstart.md]:
```suggestion
dapr run --app-id order-processor --components-path ../../../components/ -- go run .
```

File: daprdocs/content/en/getting-started/quickstarts/statemanagement-quickstart.md [hunk 5]
```diff
@@ -532,13 +532,13 @@ cd state_management/go/sdk/order-processor
 Install the dependencies and build the application:
 
 ```bash
-go build
+go build app.go
```
Comment (hhunter-ms) [inline daprdocs/content/en/getting-started/quickstarts/statemanagement-quickstart.md]:
```suggestion
go build .
```

File: daprdocs/content/en/getting-started/quickstarts/statemanagement-quickstart.md [hunk 6]
```diff
@@ -532,13 +532,13 @@ cd state_management/go/sdk/order-processor
...
 ```
 
 Run the `order-processor` service alongside a Dapr sidecar.
 
 ```bash
-dapr run --app-id order-processor --components-path ../../../components -- go run
+dapr run --app-id order-processor --components-path ../../../components -- go run app.go
```
Comment (hhunter-ms) [inline daprdocs/content/en/getting-started/quickstarts/statemanagement-quickstart.md]:
```suggestion
dapr run --app-id order-processor --components-path ../../../components -- go run .
```

File: daprdocs/content/en/getting-started/quickstarts/bindings-quickstart.md [hunk 7]
```diff
@@ -899,20 +902,20 @@ cd quickstarts/bindings/go/sdk/batch
 Install the dependencies:
 
 ```bash
-go build
+go build app.go
```
Comment (hhunter-ms) [inline daprdocs/content/en/getting-started/quickstarts/bindings-quickstart.md]:
```suggestion
go build .
```

File: daprdocs/content/en/getting-started/quickstarts/bindings-quickstart.md [hunk 8]
```diff
@@ -899,20 +902,20 @@ cd quickstarts/bindings/go/sdk/batch
...
 ```
 
 Run the `batch-sdk` service alongside a Dapr sidecar.
 
 ```bash
-dapr run --app-id batch-sdk --app-port 6002 --dapr-http-port 3502 --dapr-grpc-port 60002 --components-path ../../../components -- go run 
+dapr run --app-id batch-sdk --app-port 6002 --dapr-http-port 3502 --dapr-grpc-port 60002 --components-path ../../../components -- go run app.go
```
Comment (hhunter-ms) [inline daprdocs/content/en/getting-started/quickstarts/bindings-quickstart.md]:
```suggestion
dapr run --app-id batch-sdk --app-port 6002 --dapr-http-port 3502 --dapr-grpc-port 60002 --components-path ../../../components -- go run .
```

**Review by hhunter-ms — CHANGES_REQUESTED** [review]
updated commands to match [quickstart](https://github.com/dapr/quickstarts) readme files

**Comment by rogerogers** [comment]
done, go build . is better.

**Review by msfussell — APPROVED** [review]

**Review by msfussell — APPROVED** [review]

**Review by hhunter-ms — APPROVED** [review]

**Pull request merged**